src/components/Layout/MoviesList/MoviesList.tsx

```diff
@@ -0,0 +1,20 @@
+import { MovieItem, MovieItemProp } from '../MovieItem/MovieItem';
+import './MoviesList.scss';
+import DATA from '../../../constants/data.json';
+
+interface MoviesListProps {
+  searchEntry: string;
+}
+const MoviesList = (props: MoviesListProps): JSX.Element => {
+  console.log(props.searchEntry);
+  const movies: MovieItemProp[] = DATA.data;
+
+  if (movies) {
+    const card = movies.map((movie) => <MovieItem {...movie} key={movie.id} />);
+    return <div className="mwMoviesList">{card}</div>;
+  } else {
```

Comment:
To avoid `else` you could, again, use negation - it is much more readable, as my experience suggests.

Reply:
I think it depends. for me it is more easier to read when I see else block.